app/src/sharedTest/java/org/oppia/android/app/devoptions/MarkChaptersCompletedActivityTest.kt

```diff
@@ -0,0 +1,462 @@
+          position = 3,
+          targetViewId = R.id.mark_chapters_completed_chapter_summary_recycler_view
+        )
+      ).check(matches(isDisplayed()))
+      scrollToPosition(4)
+      onView(
+        atPositionOnView(
+          recyclerViewId = R.id.mark_chapters_completed_story_summary_recycler_view,
+          position = 4,
+          targetViewId = R.id.mark_chapters_completed_chapter_summary_recycler_view
+        )
+      ).check(matches(isDisplayed()))
+    }
+  }
+
+  @Test
+  fun testMarkChaptersCompletedActivity_configChange_chapterListSAreVisible() {
+    launch<MarkChaptersCompletedActivity>(
+      createMarkChaptersCompletedActivityIntent(
+        internalProfileId = internalProfileId
+      )
+    ).use {
+      testCoroutineDispatchers.runCurrent()
+      onView(isRoot()).perform(orientationLandscape())
+      scrollToPosition(0)
+      onView(
+        atPositionOnView(
+          recyclerViewId = R.id.mark_chapters_completed_story_summary_recycler_view,
+          position = 0,
+          targetViewId = R.id.mark_chapters_completed_chapter_summary_recycler_view
+        )
+      ).check(matches(isDisplayed()))
+      scrollToPosition(1)
+      onView(
+        atPositionOnView(
+          recyclerViewId = R.id.mark_chapters_completed_story_summary_recycler_view,
+          position = 1,
+          targetViewId = R.id.mark_chapters_completed_chapter_summary_recycler_view
+        )
+      ).check(matches(isDisplayed()))
+      scrollToPosition(2)
+      onView(
+        atPositionOnView(
+          recyclerViewId = R.id.mark_chapters_completed_story_summary_recycler_view,
+          position = 2,
+          targetViewId = R.id.mark_chapters_completed_chapter_summary_recycler_view
+        )
+      ).check(matches(isDisplayed()))
+      scrollToPosition(3)
+      onView(
+        atPositionOnView(
+          recyclerViewId = R.id.mark_chapters_completed_story_summary_recycler_view,
+          position = 3,
+          targetViewId = R.id.mark_chapters_completed_chapter_summary_recycler_view
+        )
+      ).check(matches(isDisplayed()))
+      scrollToPosition(4)
+      onView(
+        atPositionOnView(
+          recyclerViewId = R.id.mark_chapters_completed_story_summary_recycler_view,
+          position = 4,
+          targetViewId = R.id.mark_chapters_completed_chapter_summary_recycler_view
+        )
+      ).check(matches(isDisplayed()))
+    }
+  }
+
+  @Test
+  fun testMarkChaptersCompletedActivity_selectAll_isChecked() {
+    launch<MarkChaptersCompletedActivity>(
+      createMarkChaptersCompletedActivityIntent(
+        internalProfileId = internalProfileId
+      )
+    ).use {
+      testCoroutineDispatchers.runCurrent()
+      onView(withId(R.id.mark_chapters_completed_all_check_box))
+        .perform(click()).check(matches(isChecked()))
+    }
+  }
+
+  @Test
+  fun testMarkChaptersCompletedActivity_selectAll_thenDeselect_isNotChecked() {
```

Comment:
Found something interesting about `Select-Deselect`. 

Google Defination
```
Deselect is the real antonym of select. Unselect is only used when talking about computers. 
```

Also, no need of `then`

Reply:
replaced `deselect` with `unselect`. also removed `then`